### DISCUSSION TITLE
Self Hosted Supabase & PgBouncer

### DISCUSSION THREAD

**Discussion by kevinmlong** [supabase/supabase#12340] [Questions]
I'm self hosting Supabase and was wondering if the port specified for Postgres on the various containers should be the PgBouncer port or the normal Postgres port. I've NNO ticked that sometimes we're seeing connection errors and I got to thinking about the connection pooling and if this might be why.

**Comment by GaryAustin1** [comment]
If you are not using the direct port for access to the database from clients then probably not.  PostgREST has its own pooler in it and you don't want it to go thru the bouncer port (or at least with out lots of extra work that is probably not needed).
